core/src/main/java/nl/nn/adapterframework/jms/XmlJmsBrowserSender.java

```diff
@@ -99,7 +103,14 @@
  * 
  * @author  Peter Leeuwenburgh
  */
-public class XmlJmsBrowserSender extends SenderWithParametersBase {
+public class XmlJmsBrowserSender extends SenderWithParametersBase implements ApplicationContextAware {
+
+	public @Setter ApplicationContext applicationContext = null;
+
+	@SuppressWarnings("unchecked")
+	public JmsBrowser<javax.jms.Message> createJmsBrowser() {
+		return (JmsBrowser<javax.jms.Message>) applicationContext.getAutowireCapableBeanFactory().createBean(JmsBrowser.class, AutowireCapableBeanFactory.AUTOWIRE_BY_NAME, false);
```

Comment:
Is het niet handig om SenderBase, net als AbstractPipe, een eigen createBean() te geven? Ter opsimpeling van dit soort code?